de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/table/api/impl/CreateColumnsProducer.java

```diff
@@ -0,0 +1,210 @@
+package org.adempiere.ad.table.api.impl;
```

Comment:
move it near the process which is using it. change access to "package" level. I doubt somebody else will use it.

More, it's not documented. Nobody knows what the "input stream"  shall contain, what format etc.
My fear is that process is one time only, but we can live with that now.

Next time, pls consider creating the table in database and then importing it to application dictionary.
See, the syntax is almost as simple as the file which is expected here: https://www.postgresql.org/docs/9.5/static/sql-createtable.html

Reply:
Added some java docs. But I will delete the process in the end. I use it because was easier to copy columns from the given pdf file.